core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -139,8 +150,33 @@ class HttpGoogleServicesDAO(
           setAcl(bucketAcls).
           setDefaultObjectAcl(defaultObjectAcls)
         val insertMetadataBucket = getStorage(getBucketServiceAccountCredential).buckets.insert(serviceProject, metadataBucket)
+        Await.result(newGoogleStorage.setBucketLifecycle(cromwellMetadataBucketName, 30), 5 seconds)
         executeGoogleRequest(insertMetadataBucket)
     }
+
+    // unsafeRunSync is not desired, but return type for initBuckets dictates execution has to be happen immediately when the method is called.
+    // Changing initBuckets's signature requires larger effort which doesn't seem to worth it
+    createTopic.unsafeRunSync()
```

Comment:
I'm a bit confused here and need a bit of clarification - I thought there would be 1 topic that rawls would publish to. This seems like it's creating a new topic every time this class is initialized. Is this what we want?

Reply:
Your assumption is right. `createTopic` is trying to creating the same topic every time, and if the topic already exists, google gives us an exception, and we ignore the error. Why do you think there's new topic created every time?